python/tk_multi_launchapp/prepare_apps.py

```diff
@@ -32,23 +33,24 @@ def prepare_launch_for_engine(engine_name, app_path, app_args, context, file_to_
     # Retrieve the TK Application instance from the current bundle
     tk_app = sgtk.platform.current_bundle()
 
-    try:
-        # Try to use the TK engine to perform the necessary preparations
+    # Make sure this version of core supports the create_engine_launcher method, this was introduced
+    # very recently, but we don't want to lock bugfixes to the legacy launch system behind a core upgrade.
+    if hasattr(sgtk.platform, "create_engine_launcher"):
+        # Use the TK engine to perform the necessary preparations
         # to launch the DCC. If launcher is None, then chances are the
         # installed version of the specified engine isn't up-to-date.
         launcher = sgtk.platform.create_engine_launcher(
             tk_app.sgtk, context, engine_name
         )
-        tk_app.log_debug("Created %s engine launcher : %s" %
-            (engine_name, launcher)
-        )
         if launcher:
+            tk_app.log_debug("Created %s engine launcher : %s" % (engine_name, launcher))
```

Comment:
There is one, it's right [here](https://github.com/shotgunsoftware/tk-multi-launchapp/pull/45/files#r103786725)